src/wx/panel.cpp

```diff
@@ -80,13 +80,13 @@ void GameArea::LoadGame(const wxString& name)
 
     // auto-conversion of wxCharBuffer to const char * seems broken
     // so save underlying wxCharBuffer (or create one of none is used)
-    wxCharBuffer fnb(fnfn.GetFullPath().mb_fn_str());
+    wxCharBuffer fnb(fnfn.GetFullPath().mb_str(wxConvUTF8));
```

Comment:
If I understand the situation correctly, we have an issue of ping-ponging a string between UTF-8, UTF-16 and whatever else Windows sends in our way. `wxString` can deal with these, but we needed to use across all files, which is not true since we have `const char *` and `wchar_t *` on multiple files. This is a way that I find on [docs](https://docs.wxwidgets.org/3.0/classwx_string.html#adcfd12e6d0765b1d74bccc3d63d02e98) to get a `UTF-8` string converted automatically and use as an input for `utilFindType`.